recipe/meta.yaml

```diff
@@ -105,7 +105,9 @@ test:
   # See: https://github.com/man-group/ArcticDB/blob/master/environment_unix.yml
   # See `options.extras_require`: https://github.com/man-group/ArcticDB/blob/master/setup.cfg
     - pytest
-    - pyarrow
+    # pyarrow cannot be resolved with the current constraints of the environment
+    # for some reasons which need investigation.
+    # - pyarrow
```

Comment:
I also started getting pyarrow related conda solver errors in a test environment this week (started Monday night). They finally started solving again this afternoon. I suspect the solver error is related to recent conda-forge migrations (eg libabseil). I recommend adding it back to see if it solves now.

xref: https://github.com/TileDB-Inc/conda-forge-nightly-controller/issues/193

Reply:
https://github.com/conda-forge/arcticdb-feedstock/pull/476 needs to be merged first since https://github.com/conda-forge/arrow-cpp-feedstock/pull/1787 has been merged before https://github.com/conda-forge/arrow-cpp-feedstock/pull/1794 (but currently https://github.com/conda-forge/arcticdb-feedstock/pull/476#issuecomment-3051395973).